view/frontend/templates/checkout/onepage/success/tesco_additional_info.phtml

```diff
@@ -1,5 +1,5 @@
 
 <p><?= __('To finish payment, print the code below and bring it to make payment in any Tesco Lotus Store. Code is valid for the next <b>24 hours only</b>. After that time your order will be canceled.'); ?></p>
 <p style="margin:.5rem 0"><?= $block->escapeHtml(__('Amount to pay'))?>: <strong><?= $block->getOrderAmount();?></b></strong>
-<p><img src="<?= $block->escapeUrl($block->getTescoCodeUrl()) ?>" /></p>
+<div><?= $block->getTescoCodeImage() ?></div>
```

Comment:
@jacstn just wanna point out that actually looking at this change without understanding the context seems a bit strange. 
While the code is changed to `getTescoCodeImage()` but we removed `<img></img>` tag.

Reply:
after digging deeper into code, we still display in HTML - SVG code.

SVG is converted to HTML only when it is necessary - before sending it in email.

`img` with `src` was necessary when we were using `URL` to `SVG` file.
I renamed it to `getTescoBarcodeSvg` and corresponding array key to: `tesco_barcode_svg`